Personalizing your Skywallet.md

```diff
@@ -1,22 +1,50 @@
 
-The list can be viewed when you open your Desktop Wallet.
+<can be removed> <Screenshot of the list of Wallets displayed when the Desktop Wallet is opened. One of the Wallets in the list needs to be a Skywallet>
 
-<Screenshot of the list of Wallets displayed when the Desktop Wallet is opened. One of the Wallets in the list needs to be a Skywallet>
+To personalize your Skywallet, connect your Skywallet via USB to your computer and then open the Desktop Wallet.
 
-To personalize your Skywallet, you have to click on the **Hardware Wallet** button.
+Once the Desktop Wallet is opened, click on the **Hardware Wallet** button in the Desktop Wallet window.
 
-<Screenshot of the Desktop Wallet highlighting the Hardware Wallet button>
+<can be removed> <Screenshot of the Desktop Wallet highlighting the Hardware Wallet button>
 
 Once you click on the "Hardware Wallet" button, the following window will appear:
 
 <Screenshot of the Hardware Wallet window with the options available when the Skywallet is used after configuration for the first time>
 
-When you use the Skywallet for the first time, you will find these two warnings associated with **Creating a Backup** and **Setting a Pin** being displayed in the Desktop Wallet window.
+When you use the Skywallet for the first time, you will find two warnings associated with **Creating a Backup** and **Setting a Pin**, displayed in the Desktop Wallet window.
+
+Both of these warnings are to be dealt with seriously, or it may compromise the security of your Skywallet.
+
+To complete these two tasks, you can use the options in the Hardware Wallet window.
+
+Following are the options available in this window:
+
+**Create a backup**
+
+**Create PIN code**# Personalizing your Skywallet
+
+After completing the initialization of your Skywallet, you can start to personalize your Skywallet using the Desktop Wallet.
+
+Upon configuring your Skywallet, it will be listed in the Wallet list. The list can be viewed when you open your Desktop Wallet.
+
+<can be removed> <Screenshot of the list of Wallets displayed when the Desktop Wallet is opened. One of the Wallets in the list needs to be a Skywallet>
+
+To personalize your Skywallet, connect your Skywallet via USB to your computer and then open the Desktop Wallet.
+
+Once the Desktop Wallet is opened, click on the **Hardware Wallet** button in the Desktop Wallet window.
+
+<can be removed> <Screenshot of the Desktop Wallet highlighting the Hardware Wallet button>
+
+Once you click on the "Hardware Wallet" button, the following window will appear:
+
+<Screenshot of the Hardware Wallet window with the options available when the Skywallet is used after configuration for the first time>
+
+When you use the Skywallet for the first time, you will find two warnings associated with **Creating a Backup** and **Setting a Pin**, displayed in the Desktop Wallet window.
```

Comment:
The same line exists earlier in this text.

Reply:
This was some copy paste issue the first 28 odd lines where repeated, have removed it.